src/puppetlabs/puppetdb/random.clj

```diff
@@ -1,38 +1,35 @@
 (ns puppetlabs.puppetdb.random
   (:require
    [clojure.string :as string]
-   [clojure.walk :refer [keywordize-keys]]))
+   [clojure.walk :refer [keywordize-keys]])
+  (:import
+   (org.apache.commons.lang3 RandomStringUtils)))
+
+;; FIXME: https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/RandomStringUtils.html
+
+(set! *warn-on-reflection* true)
```

Comment:
Should we remove this warn-on-reflections setting?

And should we remove the FIXME above as well, or provide some more context for what needs to be fixed?

Reply:
Thanks.  Both vestigial - removed.